soccer/planning/IndependentMultiRobotPathPlanner.cpp

```diff
@@ -33,6 +35,14 @@ std::map<int, std::unique_ptr<Path>> IndependentMultiRobotPathPlanner::run(
         staticRobotObstacles[shell] = std::make_shared<Geometry2d::Circle>(
             request.start.pos, Robot_Radius);
     }
+
+    auto comparator = [&](int &lhs, int &rhs) {
+            return sortByPriority(requests[lhs], requests[rhs]);
```

Comment:
It would be a little shorter if you get rid of `sortByPriority()` and just do `return requests[lhs].priority > requests[rhs].priority` here.